Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -737,6 +738,67 @@ class AEPEdgeFunctionalTests: FunctionalTestBase {
 
         self.assertNetworkRequestsCount()
     }
+
+    func testSendEvent_multiStatusResponse_dispatchesEvents() {
```

Comment:
Can we include one test with fatal error too with error details at the top level in the json.